pkg/machinery/go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/talos-systems/net v0.3.2
 	go4.org/netipx v0.0.0-20220925034521-797b0c90d8ab
 	google.golang.org/genproto v0.0.0-20220930163606-c98284e70a91
-	google.golang.org/grpc v1.50.0
```

Comment:
go.mod/go.sum updates are move complicated: (private link) https://www.notion.so/siderolabs/Updating-Talos-Go-modules-1bad3ac587e94d80b6766f87549b2b71

Reply:
On it now, thanks.